packages/grafana-data/src/events/common.ts

```diff
@@ -0,0 +1,45 @@
+import { DataFrame } from '../types';
+import { eventFactory } from './eventFactory';
+import { BusEventWithPayload } from './types';
+
+/** @alpha */
+export interface DataHoverPayload {
+  raw: any; // Original mouse event (includes pageX etc)
+
+  x: Record<string, number>; // { time: 5678 },
+  y: Record<string, number>; // { __fixed: 123, lengthft: 456 }  // each axis|scale gets a value
```

Comment:
always forcing numbers on these, wont that be a problem for BarChart / categorical x

Reply:
switched to any